packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -1501,6 +1504,48 @@ abstract class FlutterCommand extends Command<void> {
     );
   }
 
+  // This adds the Dart defines used to access various Flutter version information at runtime.
+  void _addFlutterVersionToDartDefines(FlutterVersion version, List<String> dartDefines) {
+    const String flutterVersionDefine = 'FLUTTER_VERSION';
```

Comment:
I think we should move these to be static constants on the `FlutterCommand` class (maybe marked `@visibleForTesting`). That way, we can use `flutterVersionDartDefines` in tests rather than recreating this list of define keys that could fall out of sync.

Reply:
Done